identity/src/main/java/com/stripe/android/identity/ui/ConfirmationScreen.kt

```diff
@@ -53,92 +52,90 @@ internal fun ConfirmationScreen(
     val verificationPageState by identityViewModel.verificationPage.observeAsState(Resource.loading())
     val context = LocalContext.current
 
-    MdcTheme {
```

Comment:
Note: all Screen changes are just removing the encapsulating `MdcTheme`, as we now have it in the NavHost